root/static/scripts/relationship-editor/components/RelationshipEditor.js

```diff
@@ -556,7 +556,7 @@ type ErrorMessagePropsT = {
 };
 
 export const ErrorMessage:
-  React.AbstractComponent<ErrorMessagePropsT, mixed> =
+  component(ref: React.RefSetter<mixed>, ...ErrorMessagePropsT) =
```

Comment:
Isn't `mixed` the default?

Reply:
It seems that it's necessary to specify a `ref` on the component in order for it to be compatible with `React.memo`'s `Instance = mixed`: https://gist.github.com/mwiencek/015208b73f12cf956abbdbbf24b5bedc